src/uct/api/uct.h

```diff
@@ -437,6 +437,19 @@ enum {
 
 
 /**
+ * @ingroup UCT_RESOURCE
+ * @brief   @anchor UCT_FLUSH_FLAGS Flags that control completion semantic.
+ */
+enum {
+    UCT_FLUSH_FLAG_LOCAL = 0,           /**< Guarantees flush of local
```

Comment:
if you do not have objections, let's define all APIs in one place then I split the PRs.